bigquery/snippets/bqtestutil/bqtestutil.go

```diff
@@ -52,3 +53,18 @@ func sanitize(s string, allowedSeparator string) string {
 	}
 	return reg.ReplaceAllString(s, "")
 }
+
+// RunCMEKTests probes whether CMEK-based tests should run.
+func RunCMEKTests() bool {
+	// TRAMPOLINE_BUILD_FILE is set by the CI testing we use, and is a quick
+	// heuristic for testing whether this is a CI-based build.
+	_, ok := os.LookupEnv("TRAMPOLINE_BUILD_FILE")
```

Comment:
Would you mind using another variable? We might disable Trampoline at some point.

Reply:
Switched this to one of the default variables KOKORO_BUILD_ID